citadel/indico_citadel/templates/event_category_warning.html

```diff
@@ -0,0 +1,9 @@
+{% from 'message_box.html' import message_box %}
+
+{% call message_box('warning', large_icon=true, fixed_width=true) %}
+    {% trans category_title=category.title, max_events_threshold=threshold -%}
+        <b>The '{{ category_title }}' category contains over {{ max_events_threshold }} events - this may cause performance issues.</b>
+        <br/>
+        Please consider using groups instead of individual users to avoid having to re-send large amounts of changes to users individually.
```

Comment:
The category title is already shown on the page, no need to repeat it.

I would show something simple like this:

> **This category contains a large number of events.**
> Please consider using groups instead of individual users when granting access or management privileges since any change to the access control list requires re-synchronizing all events with Indico's search engine.

(not 100% sure about the second sentence though)

Reply:
I think this sounds pretty concise (and it can be changed later on).